serviceAccessConfig/lib/serviceAccessConfig/apache.py

```diff
@@ -126,6 +126,16 @@ def _update_service_config(self, cidr_blocks):
                         found_order = 1
                         if ip_directive == 'Allow from':
                             new_content += order
+                    # Strip opening and closing directive from new
+                    # access restriction list if they exist and are in the
+                    # config file
+                    elif ( ln.startswith(opening_directive) ):
```

Comment:
Just to be complete, consider checking that it ends with `closing_directive` as well?

Reply:
Well closing_directive would be on another line which implies I'd have to set some flag, to result in

if openning_directive_found and ln.strip().startswith(closing_directive):
    allow = .....

I don't think this is necessary because if the closing directive is not there in the first place then Apache would not run and complain a syntax error in the config.